### DISCUSSION TITLE
support for remote hosts via SSH?

### DISCUSSION THREAD

**Discussion by patricegautier** [containrrr/watchtower#1079] [Feature Requests] [answered]
Hi All - poking around, I was hoping to connect one Watchtower instance to a remote host via SSH.

Looks like that is not supported, correct me if I'm wrong, only the tcp:// method with certs.

Is there any special reason that is, or just not implemented?

**Comment by simonaronsson** [comment, marked as answer]
Mainly, it's just not implemented, but also - why wouldn't you just deploy a watchtower instance on this remote host instead? The use case seems a bit too fringe to motivate spending time implementing it.

**Comment by patricegautier** [comment]
That is what I ended up doing.  I was looking to centralize the watchtower across machines, but it certainly is easy to work around.

Remote access via certs is implemented so someone thought remote access was worth it! :)
